### ISSUE TITLE
user shoudn't be able to move to results if he was the only voter

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-core#54]
I was the only one to vote at http://test.vilfredo.org/question/6/results and then clicked on Move to Results. I won.

**Comment by pietrosperoni** [comment]
maybe. What if instead it opens a window saying: "you were the only person voting, as such the result is only valid to let you understand what do you prefer. If instead you prefer to know what other people prefer, you should wait for them to vote as well (and maybe ping them)". After all vilfredo could be used by a single person to reorder his priorities

**Comment by pietrosperoni** [comment]
I would say this depends upon how many people were invited to the question. Generally if n people were invited to the question and m people voted, with m<n, then the system should open up an alert box before moving on. And giving instead the option to ping the late people, or exclude them

**Comment by giovanisp** [comment]
Right. v2.2 ?

**Comment by pietrosperoni** [comment]
yes, probably. 2.2 and in case we move it on at that point to 2.3
